lib/lhm/chunker.rb

```diff
@@ -55,12 +55,12 @@ def copy_batchwise(select_clause)
 
     def select_start
       start = connection.select_value("select min(#{origin_primary_key}) from #{ origin_name }")
-      start ? start.to_i : nil
+      start ? start.to_i : 1
```

Comment:
When a table has 0 rows, we are getting the below exception while processing the chunks, hence modified the fallback value as 1 to handle this.

`
NoMethodError: undefined method `-' for nil:NilClass

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm/chunker.rb:120:in `execute_batch_mode'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm/chunker.rb:103:in `execute'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm/command.rb:17:in `run'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm/invoker.rb:40:in `block in run'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm/command.rb:14:in `run'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm/invoker.rb:39:in `run'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/bundler/gems/large-hadron-migrator-32ca5674641e/lib/lhm.rb:47:in `change_table'

/Users/tradhakrishnan/Documents/GitHub/freshsales/db/migrate/20230810042927_add_association_id_scan_status_to_documents.rb:13:in `up'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/gems/activerecord-4.2.11.3/lib/active_record/migration.rb:611:in `exec_migration'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/gems/activerecord-4.2.11.3/lib/active_record/migration.rb:592:in `block (2 levels) in migrate'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/gems/activerecord-4.2.11.3/lib/active_record/migration.rb:591:in `block in migrate'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/gems/activerecord-4.2.11.3/lib/active_record/connection_adapters/abstract/connection_pool.rb:292:in `with_connection'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/gems/activerecord-4.2.11.3/lib/active_record/migration.rb:590:in `migrate'

/Users/tradhakrishnan/.rvm/gems/ruby-2.3.7@rails3.2/gems/active_record_shards-3.17.0/lib/active_record_shards/migration.rb:108:in `migrate_with_forced_shard'

/Users/tradhakrishnan/Documents/GitHub/freshsales/config/initializers/activerecord_patches.rb:68:in `migrate'
`